employee_project/employee_class.py

```diff
@@ -0,0 +1,215 @@
+
+import os
+import re
+from Constants import *
+emp_id = 0
+employee_database = dict()
+
+
+class Employee_info():
+    """
+    My Employee_info class works get the employee details and add the employee details and delete the employee details and
+    display the employee details
+    """
+    def __init__(self,name, age, phone_number, address, designation, salary):
+        """
+        Constructor For  Employee_info Class .
+
+        Args:
+            arg1 (str): The first argument .
+            arg2 (int): The second argument.
+            arg3 (int): The threed argument.
+            arg4 (str): The fourth argument.
+            arg5 (str): The fourth argument.
+            argr (int): The fourth argument.
+
+        Return :
+            None
+        """
+        self.emp_name = name
+        self.emp_age = age
+        self.emp_phone = phone_number
+        self.emp_address = address
+        self.emp_designation = designation
+        self.emp_salary = salary
+
+    def emp_details(self):
+        """ @param: str
+            @desc: Display The Employee Details
+            @returns: None
+        """
+        print(" Employee Name: ", self.emp_name)
+        print(" Employee Age: ", self.emp_age)
+        print(" Employee Phone Number: ", self.emp_phone)
+        print(" Employee Address: ", self.emp_address)
+        print(" Employee Designation: ", self.emp_designation)
+        print(" Employee Salary: ", self.emp_salary)
+
+
+    def get_employee_name(self,employee_name):
+        """ @param: str
+            @desc: add name to Employee Details
```

Comment:
Get the input from the user and Verifying the user input is proper or not , Return the Employee Name